docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -6,12 +6,18 @@
 @sphinxdirective
 
 
-The CPU plugin is a part of the Intel® Distribution of OpenVINO™ toolkit. It is developed to achieve high performance inference of neural networks on Intel® x86-64 CPUs.The newer 11th generation and later Intel® CPUs provide even further performance boost, especially with INT8 models.
+The CPU plugin is a part of the Intel® Distribution of OpenVINO™ toolkit. It is developed to achieve high performance inference of neural networks on Intel® x86-64 and Arm® CPUs. The newer 11th generation and later Intel® CPUs provide even further performance boost, especially with INT8 models.
 For an in-depth description of CPU plugin, see:
 
 - `CPU plugin developers documentation <https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/cmake_options_for_custom_compilation.md>`__.
 - `OpenVINO Runtime CPU plugin source files <https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/>`__.
 
+.. note::
+   Arm® platform is supported on a community level. Intel® welcomes community participation in the OpenVINO™ ecosystem, 
```

Comment:
This note seems to be not relevant anymore. I would keep only the following statement: `The scope of the CPU plugin features and optimizations on Arm® may differ from Intel® x86-64.`  We can also extend desclaimer with smt like: `if limitation is not mentioned explicitly it means the feature is supported for all CPU acrhitectures`